LawsonMcCoy/Fading_Light_Narcissus_Lux
# Cloud Import

TESTING INSTRUCTIONS:

1. There will be a new scene called "CloudTestScene" under Scenes->TestScenes. Open the Scene.
2. Make sure that the Cloud Game Object is in the scene, and is rendering properly.
3. Make sure that the Cloud Game Object is also a prefab under the Prefabs folder.
4. ADDITIONAL: I included this Cloud Game Object on our current Playground Tutorial Scene just to see how it looks; feel free to check that scene out as well.
